### ISSUE TITLE
Skills don't improve

### ISSUE THREAD

**Issue by ArisHard** [arcadia-redux/custom_hero_clash_issues#2371]
Улучшил дальность атаки , но не получилось. Подумал случайность и забыл. На днях улучшил урон который в процентах , также не получилось. Сейчас стоит на улучшения весь магический урон , уверен что тоже не получится

**Comment by ArisHard** [comment]
Improved the attack range, but it didn't work out. I thought it was an accident and forgot. The other day I improved the damage, which is a percentage, also failed. Now it 's worth improving all the magic damage , I 'm sure it won 't work either

**Comment by ZLOY5** [comment]
Это не полоска прогресса улучшения, это срок его действия, улучшения 2 и 3 уровней покупается на 30 дней. Мы же про вехи говорим?

**Comment by ArisHard** [comment]
> Это не полоска прогресса улучшения, это срок его действия, улучшения 2 и 3 уровней покупается на 30 дней. Мы же про вехи говорим?

Yes, about milestones. 30 days have passed, but there is no improvement. Right now I'm going to look at the improvements for all the damage from spells.

**Comment by ArisHard** [comment]
Да , про вехи. 30 дней прошло , но улучшения нет. Щас зайду посмотрю улучшения на весь урон от заклинаний.

**Comment by ZLOY5** [comment]
Скриншот сделай

**Comment by ArisHard** [comment]
![image](https://user-images.githubusercontent.com/117206896/200523527-0389f012-c9e6-4949-8192-c092ace61af1.png)
![image](https://user-images.githubusercontent.com/117206896/200523626-49478bd6-ee4f-4cb3-be47-881a7276c431.png)
![image](https://user-images.githubusercontent.com/117206896/200523667-70143a50-270c-49fb-b589-bcddad56e0d1.png)

**Comment by ZLOY5** [comment]
Вот там где написано Осталось: 3д это значит что через 3 дня это улучшение пропадет и его нужно будет заново купить. Улучшение покупается на 30 дней

**Comment by ArisHard** [comment]
Аааа...Вот оно как , я думал это ждать 30 дней надо) Понял , спс
